actionpack/test/controller/integration_test.rb

```diff
@@ -57,33 +57,63 @@ def test_get_via_redirect
     @session.get_via_redirect(path, args, headers)
   end
 
+  def test_get_via_redirect_is_deprecated
```

Comment:
Instead of adding new tests, you can wrap `assert_deprecated` around existing tests. I think this will also prevent the existing tests from spamming deprecation text in the test log.